pulsar-opentelemetry/src/main/java/org/apache/pulsar/opentelemetry/OpenTelemetryAttributes.java

```diff
@@ -112,6 +112,17 @@ public interface OpenTelemetryAttributes {
      */
     AttributeKey<String> PULSAR_CLIENT_VERSION = AttributeKey.stringKey("pulsar.client.version");
 
+    AttributeKey<String> PULSAR_CONNECTION_RATE_LIMIT_OPERATION_NAME =
+            AttributeKey.stringKey("pulsar.connection.rate_limit.operation.name");
+    enum ConnectionRateLimitOperationName {
```

Comment:
nit: I think we can make these enums extends 

interface Attributes{

Attributes getAttriibutes();
}
Then, we can just pass enum itself and call getAttributes() internally.

Reply:
As discussed offline, we will not make this refactoring here.